app/javascript/components/public_health/actions/UpdateAssignedUser.js

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+import { Button, Modal, Form } from 'react-bootstrap';
+import _ from 'lodash';
+import axios from 'axios';
+
+import InfoTooltip from '../../util/InfoTooltip';
+import reportError from '../../util/ReportError';
+
+class UpdateAssignedUser extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      assigned_user: '',
+      initial_assigned_user: undefined,
+      apply_to_household: false,
+      loading: false,
+    };
+  }
+
+  componentDidMount() {
+    const distinctAssignedUser = [...new Set(this.props.patients.map(x => x.assigned_user))];
+
+    var state_updates = {};
+    if (distinctAssignedUser.length === 1 && distinctAssignedUser[0] !== null) {
+      state_updates.initial_assigned_user = distinctAssignedUser[0];
+      state_updates.assigned_user = distinctAssignedUser[0];
+    }
+
+    if (Object.keys(state_updates).length) {
+      this.setState(state_updates);
+    }
+  }
+
+  handleChange = event => {
+    event.persist();
+    if (
+      event.target.id === 'assigned_user_input' &&
+      (event.target.value === '' ||
+        (event.target.value && !isNaN(event.target.value) && parseInt(event.target.value) > 0 && parseInt(event.target.value) <= 9999))
+    ) {
+      this.setState({ assigned_user: event.target.value ? parseInt(event.target.value) : '' });
+    } else if (event.target.id === 'apply_to_household') {
+      this.setState({ apply_to_household: event.target.checked });
+    }
+  };
+
+  submit = () => {
+    let idArray = this.props.patients.map(x => x['id']);
+    let diffState = Object.keys(this.state).filter(k => _.get(this.state, k) !== _.get(this.origState, k));
+
+    this.setState({ loading: true }, () => {
+      axios.defaults.headers.common['X-CSRF-Token'] = this.props.authenticity_token;
+      axios
+        .post(window.BASE_PATH + '/patients/bulk_edit/assigned_user', {
+          ids: idArray,
+          assigned_user: this.state.assigned_user,
+          apply_to_household: this.state.apply_to_household,
+          diffState: diffState,
+        })
+        .then(() => {
+          location.href = window.BASE_PATH;
+        })
+        .catch(error => {
+          reportError(error);
+          this.setState({ loading: false });
+        });
+    });
+  };
+
+  render() {
+    return (
+      <React.Fragment>
+        <Modal.Body>
+          <div className="mt-1 mb-3">
+            Please input the desired Assigned User to be associated with all selected monitorees:
+            <InfoTooltip tooltipTextKey="assignedUser" location="right"></InfoTooltip>
+          </div>
+          <Form.Control
```

Comment:
I think we should/need to include the datalist which contains all the existing assigned users for suggestions. I think you might be able to pass that list as a prop from PatientsTable, since that information is already provided there and gets updated with each query. Let me know if you need help doing that or if that doesn't make sense!

Reply:
I did consider trying to have that datalist available in the situation. From my understanding, while in a monitoree's profile, the datalist is populated based on the existing Assigned Users that are available in the monitoree's Assigned Jurisdiction. But with the bulk update, a user could possibly select monitorees that have different Assigned Jurisdictions and thus some monitorees in a sub-jurisdiction would possibly return a paired down datalist results in comparison to monitorees in a upper-level jurisdiction. 

So given this possible situation, should I still work to include this? And if I should, how would you suggest it be populated? Based the selected monitoree that has the 'highest' level for their Assigned Jurisdiction?

Reply:
I think it would work the same way as the assigned user search works on the dashboard currently (note how you can select "All" or "Exact" in the jurisdiction selection on the left, which filters the list exactly as you described):
<img width="604" alt="Screen Shot 2020-11-19 at 7 06 10 PM" src="https://user-images.githubusercontent.com/14923551/99739234-4bcd2d80-2a9a-11eb-9262-3193c94e78ca.png">

However, I think this may be a "nice to have", but not essential. I would discuss with the PO team member assigned to this ticket.

Reply:
Was able to get this in with my latest push.